configure.ac

```diff
@@ -461,7 +461,7 @@ AC_ARG_WITH([lustre],
         [AS_HELP_STRING([--with-lustre=path],
                         [set path to lustre headers or built sources dir, default is /usr/src/lustre-*])],
         [LUSTRE=$with_lustre],
-        [LUSTRE=/usr/src/`ls /usr/src | grep lustre- | tail -1`]
+        [LUSTRE=`ls -td /usr/src/lustre-* | head -1`]
```

Comment:
adding STDERR redirection might help to avoid unnecessary noise when Lustre sources aren't installed:
```bash
2>/dev/null ls -td /usr/src/nvidia-* | head -1
```
Otherwise it looks like this:
```bash
$ ls -td /usr/src/lustre-* | head -1
ls: cannot access '/usr/src/lustre-*': No such file or directory
```

Reply:
Thank you, Dima, for the tip! Added.